### ISSUE TITLE
Missing gem dependency for Windows

### ISSUE THREAD

**Issue by smackmybitsup** [chef/kitchen-vcenter#132]
TL;DR - `chef gem install win32-security` could either be added for Windows automatically or added to docs.

### Versions:
* Version of kitchen-vcenter: kitchen-vcenter (2.7.9)
* Version of test-kitchen: test-kitchen (2.6.0)
* Version of chef:
Chef Workstation version: 20.8.125
Chef Infra Client version: 16.4.38
Chef InSpec version: 4.22.8
Chef CLI version: 3.0.24
Chef Habitat version: 1.6.56
Test Kitchen version: 2.6.0
Cookstyle version: 6.15.5

### Platform Details

* Version of vCenter: 
* Version of ESXi:

### Scenario: 
On current Windows 10 systems...
1. chef gem install kitchen-vcenter 
2. run `kitchen create`
3. WARNING:  You don't have c:\users\<username>\appdata\local\chefdk\gem\ruby\2.7.0\bin in your PATH,
          gem executables will not run.
4. `kitchen diagnose` 
--
timestamp: 2021-01-27 16:28:35 UTC
kitchen_version: 2.9.0
plugins:
  error:
    exception: "#<Kitchen::ClientError: Could not load the 'vcenter' driver from the
      load path.cannot load such file -- win32/security>"
    message: Could not load the 'vcenter' driver from the load path.cannot load such
      file -- win32/security
    backtrace:
```
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/plugin.rb:55:in
      `rescue in load'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/plugin.rb:33:in
      `load'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/driver.rb:37:in
      `for_plugin'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/config.rb:235:in
      `new_driver'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/config.rb:248:in
      `new_instance'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/config.rb:143:in
      `block in build_instances'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/config.rb:142:in
      `map'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/config.rb:142:in
      `with_index'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/config.rb:142:in
      `build_instances'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/config.rb:116:in
      `instances'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/command.rb:109:in
      `filtered_instances'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/command.rb:139:in
      `parse_subcommand'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/command/diagnose.rb:52:in
      `load_instances'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/command/diagnose.rb:31:in
      `block in call'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/command/diagnose.rb:72:in
      `record_failure'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/command/diagnose.rb:31:in
      `call'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/cli.rb:52:in
      `perform'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/cli.rb:142:in
      `diagnose'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/thor-1.0.1/lib/thor/command.rb:27:in
      `run'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/thor-1.0.1/lib/thor/invocation.rb:127:in
      `invoke_command'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/thor-1.0.1/lib/thor.rb:392:in
      `dispatch'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/thor-1.0.1/lib/thor/base.rb:485:in
      `start'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/bin/kitchen:11:in
      `block in <top (required)>'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/lib/kitchen/errors.rb:170:in
      `with_friendly_errors'
    - C:/opscode/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/test-kitchen-2.9.0/bin/kitchen:11:in
      `<top (required)>'
    - C:/opscode/chef-workstation/bin/kitchen:358:in `load'
    - C:/opscode/chef-workstation/bin/kitchen:358:in `<main>'

```

### Steps to Reproduce:
see above.

**Comment by tas50** [comment]
This looks like an upstream issue in net-ping where they're not pushing a windows specific gem with the win32-security dependency. I've opened a PR to fix that, but we can workaround it for now in Workstation.

https://github.com/eitoball/net-ping/pull/32

**Comment by tas50** [comment]
I've merged a workaround for the net-ping issue into the Workstation gemfile and that'll ship in the next release. Ideally the net-ping ships my fix and it just works in gem installs and Workstation.
